_posts/2019-01-17-things-to-do-in-nashville.md

```diff
@@ -0,0 +1,37 @@
+In the beginning, Nashville was a small town with a big city vibe, but everyday we are getting bigger and bigger! With such a multitude of options for food, art, music, and nightlife, there is so much that could be put on this list. 
+
+As a start to beginning your Nashville experience, let us suggest just a couple places and give you more resources to find more fun things to do:
+
+### Hot Chicken Restaurants:
+* Hattie B's Hot Chicken: Three locations of just pure bliss in fried deliciously spicy chicken
+* Prince's Hot Chicken: The originator of the hot chicken craze in Nashville
+* Bolton's Hot Chicken & Fish: Another fantastic hot fish and chicken restaurant. Fun fact - it was featured on David Chang's Ugly Delicious!
+
+### Unique Museums and Monuments:
+* The Parthenon: A full scale replica of the Parthenon in Greece
+* The Frist Art Museum:  Van Gogh, Monet, Degas, and Their Times: The Mellon Collection of French Art will be the main exhibit the weekend of our conference.
+* Tootsie's and Robert's Western World: Historic country honky tonks: Tootsie's & Robert's. 
+* Country Music Hall of Fame: A museum dedicated to the history of country music
+* Lane Motor Museum: The largest European collection of cars and motorcycles in the United States.
```

Comment:
Should this be "The largest collection of European cars and motorcycles..."?

Reply:
Yes definitely